infer_structcol/run_structcol.py

```diff
@@ -89,27 +93,27 @@ def calc_refl_trans(volume_fraction, Sample, ntrajectories=300, nevents=200, see
     wavelength_sigma = sc.Quantity(np.linspace(400,1000,61), 'nm')
     # The uncertainty for the reflection fraction is taken to be 1 standard
     # deviation from the mean, and was calculated using the results of 100 identical runs.    
-    sigma_measured = np.array([1.578339786806479475e-02, 1.814049675099610806e-02, 2.263508305348480368e-02, 
-                               2.280651893165159400e-02, 2.289441072296988580e-02, 2.475289930703982594e-02, 
-                               2.591244161256863951e-02, 2.432751507610093206e-02, 2.840212103614853448e-02, 
-                               2.464656608869982696e-02, 2.535837658100221007e-02, 2.352910256218017013e-02, 
-                               2.264365724262535837e-02, 2.574192164180175851e-02, 2.546192844771695551e-02, 
-                               2.767992948024671981e-02, 2.399941085043348632e-02, 2.767133759422578734e-02, 
-                               2.759793344858079908e-02, 2.581248267951743655e-02, 2.664000919072649631e-02, 
-                               2.914272756298553688e-02, 2.549173729396642454e-02, 2.722649681737301583e-02, 
-                               2.322297011391676741e-02, 2.409138186086920430e-02, 2.807311239866464025e-02, 
-                               3.018509924866123045e-02, 2.929772336148638717e-02, 2.866675231142475078e-02, 
-                               2.377896176281297722e-02, 2.532538972626817778e-02, 2.408458082494839558e-02, 
-                               2.823887112376391451e-02, 2.285680624758363796e-02, 2.834624619602043455e-02, 
-                               2.342167374818072967e-02, 2.896504983742856365e-02, 2.835463183413225105e-02, 
-                               2.981124596936481769e-02, 2.499991827718371987e-02, 2.697080309787770400e-02, 
-                               2.788310424558666095e-02, 2.819362357263776805e-02, 2.852537757990830647e-02, 
-                               2.651641629976883227e-02, 3.022850005391930842e-02, 2.772006618991802729e-02, 
-                               2.971671988748269405e-02, 3.219841220549832933e-02, 2.570752641741474998e-02, 
-                               2.352680291863861600e-02, 2.709648629442167056e-02, 2.524674214046034038e-02, 
-                               2.758045644043585765e-02, 2.607698592177773098e-02, 2.738258841523178236e-02, 
-                               2.868487596917410412e-02, 3.176931078488830218e-02, 2.729837088883461590e-02, 
-                               2.513728413028934808e-02])
+    sigma_measured = np.array([1.860651421552072735e-02, 1.753980839818162357e-02, 1.839622738704549398e-02, 
```

Comment:
If I recall correctly, these values depend mainly on the number of trajectories. It doesn't really make sense to hard code them for the default number of either 300 or 600 since users can specify their own number of trajectories.

I think the solution is to have the code run many identical MC model calculations with the values obtained from MAP and calculate uncertainties for itself. we are already running ~ 100 walkers * 1000 steps = 100,000 calculations so adding 100 (or even 1000) more is insignificant. That way we only need to do the calculation at the relevant wavelengths.